conf/messages

```diff
@@ -106,8 +106,8 @@ coc.end-reason.RETROSPECTIVE=Applied for Marriage Allowance retrospectively
 coc.end-reason.SYSTEM=Ended by HMRC
 coc.end-reason.DEFAULT=Marriage Allowance Ended
 
-pages.form.field.description.partners-income=Confirm if your partner has an annual income of between £{0} and £{1} (or £{2} if you live in Scotland)
-pages.form.field-required.partners-income=Tell us if your partner has an annual income of between £{0} and £{1} (or £{2} if you live in Scotland)
+pages.form.field.description.partners-income=Confirm if your partner has an annual income of between £{0} and £{1}
```

Comment:
Don't we need the Welsh language file to be updated too?

Reply:
Updated